### DISCUSSION TITLE
Suggestion: Replace Project Player with

### DISCUSSION THREAD

**Discussion by DogemanMC** [ScratchAddons/ScratchAddons#5169] [Ideas] [answered]
This is a suggestion for the addon "Replace Project Player with".

**Comment by BroJac5246** [comment, marked as answer]
Replace it with what?

**Reply by DogemanMC** [reply]
What I would expect: An option in the Scartch Addons addons tab that allows you to replace the default Scratch Project Player

**Reply by DogemanMC** [reply]
with something else like TurboWarp or forkphours(?)

**Reply by DogemanMC** [reply]
Working on it?

**Reply by BroJac5246** [reply]
This already exists as an option for the `turbowarp-button` addon

**Reply by DogemanMC** [reply]
The turbowarp-button addon applies to the project ONCE, I meant the default Project Player (the current default project player is the built in standard scratch project player) that applies to ALL projects you load. (you can change this later to the built in project player at anytime.)

**Reply by DogemanMC** [reply]
Before reloading: 
![image](https://user-images.githubusercontent.com/111196211/194460415-3c9d460f-f31c-4b38-a607-c8f703bed5df.png)
After reloading: 
![image](https://user-images.githubusercontent.com/111196211/194460579-ad54f8bb-078a-45e8-86c4-1a0ce0e2d7d1.png)

It auto-sets the project back to default (scratch built-in player)project player.
EDIT: changed the 2nd image to proper image

**Reply by BroJac5246** [reply]
Is this what you're looking for?: #3985

**Reply by DogemanMC** [reply]
Yes.

**Comment by DogemanMC** [comment]
stop
